tabpfn_client/tests/unit/test_tabpfn_classifier.py

```diff
@@ -58,10 +58,6 @@ def test_init_remote_classifier(
             mock_server.endpoints.retrieve_greeting_messages.path
         ).respond(200, json={"messages": []})
 
-        mock_server.router.get(mock_server.endpoints.protected_root.path).respond(
```

Comment:
Ideally, we don't need this here.
This line used to pass because we are calling `init()` multiple times, and the subsequent `init()` will call this API.

Reply:
@SamuelGabriel

Reply:
what do you mean? we don't need the "protected" call? I think so too as almost any action will do a protected call. removing it will save latency. do you mean that?

Reply:
This API is a dummy endpoint that requires authentication. One use case is to check the user can login successfully or not. All the other API doesn't go through.